app/src/main/kotlin/com/intive/picover/common/animations/ShimmerBrush.kt

```diff
@@ -0,0 +1,47 @@
+package com.intive.picover.common.animations
+
+import androidx.compose.animation.core.RepeatMode
+import androidx.compose.animation.core.animateFloat
+import androidx.compose.animation.core.infiniteRepeatable
+import androidx.compose.animation.core.rememberInfiniteTransition
+import androidx.compose.animation.core.tween
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color
+
+@Composable
+fun ShimmerBrush(
+	showShimmer: Boolean = true,
+	targetValue: Float = 1000f,
+): Brush {
+	return if (showShimmer) {
+		val shimmerColors = listOf(
+			Color.LightGray.copy(alpha = 0.6f),
+			Color.LightGray.copy(alpha = 0.2f),
+			Color.LightGray.copy(alpha = 0.6f),
+		)
+
+		val transition = rememberInfiniteTransition(label = "")
```

Comment:
Would be better to add real label instead of empty string :D

Reply:
Done 😄